src/contractsBuild/TokenReceiver.json

```diff
@@ -29,13 +29,13 @@
   "sourceMap": "",
   "deployedSourceMap": "",
   "source": "/*\n *  receiver contract interface\n * see https://github.com/ethereum/EIPs/issues/677\n */\npragma solidity 0.4.19;\n\n\ninterface TokenReceiver {\n    function transferNotification(address from, uint256 amount, uint data) public;\n}\n",
-  "sourcePath": "/Users/petro/ether/augmint-contracts/contracts/interfaces/TokenReceiver.sol",
+  "sourcePath": "/Users/petro/ether/augmint-web/augmint-contracts/contracts/interfaces/TokenReceiver.sol",
```

Comment:
are we happy revealing local path here?

Reply:
well, that's what truffle compile does. In the future we could maybe do the build on server